RefactoringEssentials/CSharp/Converter/MethodBodyVisitor.cs

```diff
@@ -43,6 +49,14 @@ public override SyntaxList<StatementSyntax> VisitLocalDeclarationStatement(VBSyn
 				return SyntaxFactory.List<StatementSyntax>(declarations);
 			}
 
+			public override SyntaxList<StatementSyntax> VisitAddRemoveHandlerStatement(VBSyntax.AddRemoveHandlerStatementSyntax node)
+			{
+				var syntaxKind = node.Kind() == VBasic.SyntaxKind.AddHandlerAccessorStatement ? SyntaxKind.AddAssignmentExpression : SyntaxKind.SubtractAssignmentExpression;
```

Comment:
I think this test is slightly wrong which means things always count as a removal - I'll write a test and fix it.